modules/openapi-generator/src/main/resources/python/rest.mustache

```diff
@@ -181,7 +179,7 @@ class RESTClientObject(object):
                 # Pass a `string` parameter directly in the body to support
```

Comment:
This comment should probably be updated but I'm not sure I understand the last part 'when `body` argument is provided in serialized form' so I chose to let it untouched for now.

Reply:
@sarod  My "guess" is that the body can simply a string, e.g. "OK" or a boolean value (e.g. true), or just an integer, e.g. 2019.

Usually, the HTTP request body is JSON payload and I did see a few cases in which a string/number is used.